pkg/operator/target_config_reconciler.go

```diff
@@ -67,6 +68,8 @@ import (
 
 const DefaultImage = "quay.io/openshift/origin-descheduler:latest"
 const kubeVirtShedulableLabelSelector = "kubevirt.io/schedulable=true"
+const defaultPSIPath = "/proc/pressure/"
+const psiPathEnv = "PSI_PATH"
```

Comment:
New env means new API. If anything let's make it a boolean env. E.g. "DISABLE_PSI_CHECK" to limit the scope.

Reply:
Done